README.md

```diff
@@ -1,7 +1,5 @@
 # ASP.NET Core SignalR C++ Client
 
-C++ Client for ASP.NET Core SignalR.
+This project is part of ASP.NET Core. You can find samples, documentation and getting started instructions for ASP.NET Core at https://github.com/aspnet/AspNetCore
```

Comment:
nit: full stop

Reply:
Does that mean "you're missing the ending period?"

Reply:
Yes